ReportingServicesTools/Functions/CatalogItems/Set-RsSubscription.ps1

```diff
@@ -1,154 +1,142 @@
-﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
 # Licensed under the MIT License (MIT)
 
+
 function Set-RsSubscription
 {
-    <#
+  <#
         .SYNOPSIS
-            This script set a new reporting subscription.
-        
+            This script will update subscriptions piped from Get-RsSubscriptions
+
         .DESCRIPTION
-            This script set a new reporting subscription based on the info of an existing subscription (retrieved using Get-RsSubscription).
-            You can choose a specific report or pass a folder. When using a folder, the report must have the same name.
-            NOTE: A new subscriptionId will be generated.
-                    
-        .PARAMETER Path
-            Specify the path to the destination report. Can't be used with -RsFolder parameter.
-        
-        .PARAMETER RsFolder
-            Specify the folder where the destination reports exists. Can't be used with -Path parameter.
+            This script will take the custom object producted by get-RSSubscription and use the data to update the 
+            matchdata xml to either a new startdatetime, enddate, or owner.
 
-        .PARAMETER Subscription
-            A object with all subscritpion configurations. The default output of Get-RsSubscription. You must piping it to this command.
-        
         .PARAMETER ReportServerUri
             Specify the Report Server URL to your SQL Server Reporting Services Instance.
             Use the "Connect-RsReportServer" function to set/update a default value.
-        
+
         .PARAMETER Credential
-            Specify the credentials to use when connecting to the Report Server.
+            Specify the password to use when connecting to your SQL Server Reporting Services Instance.
```

Comment:
To keep the consistency, please change from "password" to "credentials". AFAIK this is the same type the other credentials.